include/fmt/ranges.h

```diff
@@ -126,16 +126,46 @@ struct is_tuple_like {
     is_tuple_like_<T>::value && !is_range_<T>::value;
 };
 
-template <size_t... Is, class Tuple, class F>
-void for_each(std::index_sequence<Is...>, Tuple &&tup, F &&f) noexcept {
+// Check for integer_sequence
+#if defined(__cpp_lib_integer_sequence) || FMT_MSC_VER >= 1910
```

Comment:
Why 1910 for MSVC? cppreference lists that it should be in VS2015, which is 1900. https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros